### DISCUSSION TITLE
【任务】2023年5-6月社区激励任务｜50,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#498] [General]
WTF Academy 社区激励任务，包括但不限于审核，改善WTF极简教程，也包括日常的PR/issue。这部分奖励优先分配给月度社区激励任务，剩余分给当月贡献者。

## 任务 1: 校对即将出版的WTF Solidity教程前30讲 （已完成）

内容：WTF Solidity教程前30讲即将出版，出版社帮忙编辑并审核了内容，现在需要我们再给出一轮审核意见。 
<img width="624" alt="截屏2023-05-15 下午3 49 36" src="https://github.com/AmazingAng/WTF-Solidity/assets/14728591/76d1c568-4718-46f8-9340-2e9ef648e562">
截止时间: 5月21日 （周六） 24点
方式: 在 google 文档上相应内容利用 comment 功能给出修改意见。
要求：

- 学习过 WTF Solidity 教程
- 是 WTF Solidity 贡献者，并加入了 WTF 出版组。
- 阅读过英文 Solidity 官方文档。

赏金：20,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领 6 讲。

认领情况:
7-10 hollaz
11-14 sdfg
15-17 Easy
1-2, 24-30 0xKaso
3-6, 18-23 0xAA

奖励已发放： https://bscscan.com/tx/0x2c7bd1ef1ce2b729a03436eae8986d4963bdebf6f8b79d5ab02aca1b314a81ce

## 任务2: 创作WTF JavaScript 12-20 讲

**要求**：
1. WTF成员 [填表](https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform)
2. 有前端经验，写过教程/总结文章，或参加过WTF HTML/CSS教程创作。
3. 需要积极参与相关讨论。

**赏金**：20,000 $PEOPLE 代币，根据工作量分配，每人最多同时认领3讲。

12. 作用域 (闭包) [0xb00lean](https://github.com/0xb00lean)
13. 基本引用类型（deep copy） [0xb00lean](https://github.com/0xb00lean)
14. 类（原型链，构造函数，Prototypes）[lesenelir](https://github.com/lesenelir)
15. 继承 [lesenelir](https://github.com/lesenelir)
16. 事件 [holazz](https://github.com/holazz)
17. DOM（可以提一下BOM） [holazz](https://github.com/holazz)
18. 异常处理 [holazz](https://github.com/holazz)
19. 单元测试 0xAA
20. ES6 [seasidejuvenile666](https://github.com/seasidejuvenile666)
21. 事件循环/任务队列（宏任务/微任务） [seasidejuvenile666](https://github.com/seasidejuvenile666)
22. 网络请求(AJAX，Fetch API，axios) [seasidejuvenile666](https://github.com/seasidejuvenile666)
23. react入门
24. vue入门

**Comment by 0xKaso** [comment]
认领24，25、26、27、28、30讲，共6讲。

**Reply by AmazingAng** [reply]
冲

**Comment by holazz** [comment]
7、8、9、10

**Comment by sdfgsdfsdf** [comment]
认领 11 12 13 14

**Reply by sdfgsdfsdf** [reply]
审核完成

**Comment by EasyChris** [comment]
认领 15、16、17

**Reply by AmazingAng** [reply]
收到

**Reply by EasyChris** [reply]
已审核完毕

**Comment by hattiepwvi** [comment]
申请认领1，2，3，4，5，6讲

**Reply by AmazingAng** [reply]
这个任务只有WTF贡献者能认领，不好意思

**Reply by hattiepwvi** [reply]
好的，收到

**Comment by hattiepwvi** [comment]
或者申请认领18，19，20，21，22，23讲

**Comment by Azleal** [comment]
没注意到任务发布，错过了😭

**Comment by holazz** [comment]
16， 17，18

**Comment by 0xb00lean** [comment]
12、13

**Comment by lesenelir** [comment]
14 15

**Comment by seasidejuvenile666** [comment]
21 22

**Reply by AmazingAng** [reply]
ok

**Reply by seasidejuvenile666** [reply]
19 20

**Comment by yanboishere** [comment]
我也

Azleal ***@***.***>于2023年5月23日 周二17:52写道：

**Comment by hattiepwvi** [comment]
申请 第23讲 react入门 哈
